free-1.md

```diff
@@ -4,65 +4,176 @@
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to describe when are the great, good, bad, and terrible
+times to meet (likely default values).  It should also be possible to
+express more complicated dependencies, such as a preference for a meeting
+time that coincides or avoids another group member, or a freedom in a 3
+hour block where only 1 block can be scheduled at a time.
+
+It should be very difficult to do perform functions unrelated to scheduling,
+such as arithmetic, nested control flow, or see implementation details.
+
+I don't think there should be anything in between (of medium difficulty).
 
 ### Related work
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well 
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+We might consider the aforementioned websites 
+[WhenIsGood](http://whenisgood.net/) and
+[When2Meet](http://www.when2meet.com), since they are computerized schedulers
+with a specific syntax (painting over certain blocks), and associated
+semantics (painted blocks are free blocks).
+
+These DSLs are quite minimalistic (i.e. when they run, they only support the
+intersection operation for the sets of times from each individual -- as far
+as I can tell).  My solution to the scheduling problem might be a bit more
+ambitious in the functionality it hopes to provide, but I will need to take
+care during design to avoid feature creep and the complexity for users that
+may entail.  Perhaps I will find that the existing websites' simplicity is
+more desirable than the benefits of the features I would hope to add.
 
 ## The Project
 This section examines whether the idea makes for a good CS 111 project.
 
-
 ### Suitability
 _If someone were to work on this project, what percentage of their time would be
 spent directly engaging in the **language** aspects of this project (e.g.,
 making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+The underlying system should not take much implementation, it could be pretty
+easily represented as a 
+[protocol buffer](https://en.wikipedia.org/wiki/Protocol_Buffers)
+for each individual.  The most interesting part of the project, and I would
+guess that it would take at least 50% of the time, is the design of how
+users should be able to input their data.
+
+The algorithms for finding an optimal schedule could take significant work
+if that were the purpose of the project, but we could provide rather modest
+combining facilities and focus on making the data entry as interesting as
+intuitive as possible.
```

Comment:
I definitely think that making data  entry being intuitive and easy would be the key to making the DSL widely used, so I would think that this is more interesting to focus on that the algorithms for optimal schedule, although that is interesting too.